### PR TITLE
workflows/check-nix-format: Improve error message

### PR DESCRIPTION
**Pull request by infinisil**
## Description of changes

Looks like the error message could be a bit clearer still: https://github.com/NixOS/nixpkgs/pull/337109#issuecomment-2311175326 :smile:

Ping @NixOS/nix-formatting 

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Redhawk18** [comment]
Who is @NixOS/nix-formatting? It should highlight to a github account.

**Comment by infinisil** [comment]
@Redhawk18 Not sure what you mean. That's a GitHub team, comprised of the people on the [Nix Formatting Team](https://nixos.org/community/teams/formatting/)

**Comment by dasJ** [comment]
@infinisil only for org members iirc :/

**Comment by eclairevoyant** [comment]
There's one other thing I wanted to mention but kept forgetting about:
If `ci/pinned-nixpkgs.json` differs between the last commit on the base branch and the base commit of the PR, we may want to suggest that the contribute fetch the base branch and rebase their PR on it

File: .github/workflows/check-nix-format.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ jobs:
 
           if (( "${#unformattedFiles[@]}" > 0 )); then
             echo "Some new/changed Nix files are not properly formatted"
-            echo "Please go to the Nixpkgs root directory, run \`nix-shell\`, then:"
+            echo "Please format them using the Nixpkgs-specific \`nixfmt\` by going to the Nixpkgs root directory, running \`nix-shell\`, then:"
```
Comment (MattSturgeon) [inline .github/workflows/check-nix-format.yml:86]: If this were single-quoted, I think the backticks wouldn't need escaping anymore.
```suggestion
            echo 'Please format them using the Nixpkgs-specific `nixfmt` by going to the Nixpkgs root directory, running `nix-shell`, then:'
```

Not important though, if you'd rather consistently use double-quotes.

**Review by MattSturgeon — APPROVED** [review]
I agree this is clearer, +1 from me 🙂

**Review by TheGiraffe3 — APPROVED** [review]

**Comment by infinisil** [comment]
Let's aim for more [optimistic merging](http://hintjens.wikidot.com/blog:106) :D

**Pull request merged**